packages/ffmpeg.cmake

```diff
@@ -99,6 +99,7 @@ ExternalProject_Add(ffmpeg
         --enable-libplacebo
         --enable-libshaderc
         --enable-libzvbi
+        --enable-bzlib
```

Comment:
why? bzlib is detect automatically. And adding bzip2 to depends will always make sure bzlib exist before ffmpeg configure